src/components/05_pages/Content/Content.js

```diff
@@ -286,6 +310,12 @@ class Content extends Component {
             )}
           </TableBody>
         </Table>
+        <Button variant="flat" onClick={this.previousHandler}>
```

Comment:
We should use the API from Material UI for the pagination links https://material-ui.com/demos/tables/#sorting-amp-selecting

Reply:
_fwiw_, `<TablePagination />` looks like it has a bunch of extra props that we can't really provide.